src/inverted-index.js

```diff
@@ -0,0 +1,163 @@
+// import fs from 'fs';
+/**
+ * Inverted-index class
+ * @class
+ */
+class InvertedIndex {
+ /**
+  * class constructor
+  * @constructor
+  */
+  constructor() {
+    this.allIndices = {};
+    this.allFiles = {};
+  }
+  /**
+   * Validate books in a file
+   * @function
+   * @param {object} books
+   * @returns {string} response
+   */
+  static isValid(books) {
+    let response = 'Invalid JSON';
+    if (books instanceof Object) {
+      Object.keys(books).forEach((book) => {
+        if (books[book].title !== undefined && books[book].text !== undefined) {
+          if (books[book].title.length !== 0) {
+            response = 'Valid JSON';
+          } else {
+            response = 'Empty JSON';
+          }
+        } else {
+          response = 'Malformed JSON';
+        }
+      });
+    }
+    return response;
+  }
+  /**
+   * Joins text and title and pushes into an Array
+   * @function
+   * @param {object} books
+   * @returns {string} result
```

Comment:
The result variable returns a string. This happens when the join method is called on the tokens after title and text are pushed into the array.